src/storage/v3/storage.cpp

```diff
@@ -506,7 +508,8 @@ ResultSchema<VertexAccessor> Storage::Accessor::CreateVertexAndValidate(
 
   auto acc = storage_->vertices_.access();
   auto *delta = CreateDeleteObjectDelta(&transaction_);
-  auto [it, inserted] = acc.insert({Vertex{delta, primary_label, primary_properties, labels, secondary_properties}});
+  auto [it, inserted] = acc.insert(
+      {Vertex{delta, primary_label, std::move(primary_properties), labels, std::move(secondary_properties)}});
```

Comment:
I made a mistake as I didn't consider that these property values cannot be moved into the property stores. Feel free to keep this as is (we can fix that later on) or revert it.